### ISSUE TITLE
MySQL Stores Procedures/ Function support

### ISSUE THREAD

**Issue by owais9211** [mevdschee/php-crud-api#826]
Does this api support sending get/post request by calling MySQL functions and retrieving data based on the code defined in the function in form of json?.

The reason i am asking is because, i earlier worked on posgres and created some functions there which i called using postgrest api via postman.

I want to do the exact same thing with mysql, does this even possible, esp. by using the php-crud-api?.

**Comment by mevdschee** [comment]
No, this is not (yet) supported, but you may take a look at this project:

https://github.com/mevdschee/php-sp-api

It is very immature, more a proof-of-concept, but it might do what you want.

Kind regards, Maurits

**Comment by Mashiane** [comment]
Hi

Can the current api.php be used with the php-sp-api functionality?

I'm looking for something where one can specify their own query string to execute via the api. What can be needed to achieve that?

Thanks.
